### PR TITLE
Add aur repo

### PR DESCRIPTION
**Pull request by attenuation**

### CODE REVIEW

**Comment by Fierthraix** [comment]
There is already the [`up`](https://aur.archlinux.org/packages/up/) package for the official binary, and the [`up-git`](https://aur.archlinux.org/packages/up-git/) package for the source version.

**Comment by attenuation** [comment]
@Fierthraix Archlinux officials encourage the use of source code package to pack,So in fact you package way not good

**Comment by attenuation** [comment]
@akavel SOrry, I can't use such tools to prove it effective,The original aur is such this. But I'm sure that I will always maintain it until he entered the official repo. about package version,I fix it now.

**Comment by attenuation** [comment]
you can click [this](https://aur.archlinux.org/packages/go-up/) to check it

File: README.md [hunk 1]
```diff
@@ -14,8 +14,13 @@ by boosting any typical **Linux text-processing utils** such as `grep`, `sort`,
 
 ## Usage
 
+
 **[Download *up* for Linux](https://github.com/akavel/up/releases/download/v0.1/up)**
 
+If you ars using Archlinux,you can use this command to install
```
[the remark this answers is not in the document]
Comment (milkmiruku) [reply README.md]: AUR helpers are not officially supported by Arch Linux. Probably the initial thing to mention would be the official method of installing something that's in the AUR - https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages

In the area of AUR helpers - opinion on what the most popular helper is shifts every year or two. `yaourt` was the third last helper I used. I would recommend an example promoting the use of `yay`. Speak to an Arch developer and they would recommend `aurutils` - https://wiki.archlinux.org/index.php/AUR_helpers

**Comment by Fierthraix** [comment]
Okay, now [`up`](https://aur.archlinux.org/packages/up/) is the official release source code, [`up-git`](https://aur.archlinux.org/packages/up-git/) is the latest source, and [`up-bin`](https://aur.archlinux.org/packages/up-bin/) is the official binary release.

**Comment by akavel** [comment]
The Arch wiki [seems to say]( https://wiki.archlinux.org/index.php/Arch_User_Repository#Rules_of_submission ) (thanks @milkmiruku for the link!):

> Submitting binaries should be avoided if the sources are available.

So, should I advertise the binary package? or maybe assume it's going to be deleted soon(ish) and highlight the "up" package instead? I'd like to pick only one if I decide to add the info.

I'd be also super grateful for some links to real world examples of how it's done in case of some "good citizen model examples" of Arch-supporting cross-distro projects & readmes. (Should I look at, I dunno, docker's readme?) TIA!

**Comment by milkmiruku** [comment]
@Fierthraix notes binary, release source and head source PKGBUILDs are available. Nice, bases certainly covered quickly there! I'd advertise all, makes things clearer from the start.

Nothing is going to get removed, AUR isn't as policed as that. Any issues with the individual PKGBUILDs can and will be raised in their comments. Not sure who's maintaining the PKGBUILDS, though they can have co-maintainers if @akavel you wish to have a hand in helping keep them updated. Ask in #archlinux-aur on Freenode for a review if you'd like, folk there are generally happy if PKGBUILD maintainers are considerate enough to check with them.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pablox-cl) [reply README.md]: The name of the AUR packages, being the source code, the binary or whatever should be enough for an Arch user.

I use `aurget` :P.

**Comment by eli-schwartz** [comment]
> Nothing is going to get removed, AUR isn't as policed as that.

`go-up` has been deleted, since it is a *duplicate* of [`up`](https://aur.archlinux.org/packages/up/).

The [`up-bin`](https://aur.archlinux.org/packages/up-bin/) has no especial reason to be deleted, even though I think it's sort of silly to want one... but apparently someone wants it, and it doesn't break any rules...

**Comment by milkmiruku** [comment]
@eli-schwartz I don't know why you're quoting me, I was referring to the three varying entries noted by @Fierthraix. A total duplicate package is out of that context.

N.b. If you can't think of a reason for having both `up` and `up-bin` - one is applicable to multiple architectures, the other doesn't require build dependencies and doesn't take so long to install.

**Pull request closed without merging**